api/index.js

```diff
@@ -48,24 +95,38 @@ fastify.get('/kamu', async (request, reply) => {
     } catch (e) {
         console.log('error', e)
     }
-    
+
 })
 
 fastify.addHook('onReady', async () => {
     console.log('booting');
-    ({libp2p, blockstore, datastore} = await initLibp2p());
+    const {client, libp2p, blockstore, datastore} = await initLibp2p();
     heliaNode = await createHelia({
         libp2p,
         blockstore,
         datastore
     })
     console.log('helia created')
     d = dagJson(heliaNode)
+
+    mapFromCidCodeToHeliaWrapper.set(IpldDagJson.code, d)
+    console.log(`IpldDagJson.code: `, IpldDagJson.code);
+    mapFromCidCodeToHeliaWrapper.set(IpldDagCbor.code, dagCbor(heliaNode))
+    console.log(`IpldDagCbor.code: `, IpldDagCbor.code);
+    // mapFromCidCodeToHeliaWrapper.set(IpldDagPB.code, dagPb(heliaNode))
+    // console.log(`IpldDagPB.code: `, IpldDagPB.code);
+
     hiCid = await d.add({"hello": "world"})
     console.log(hiCid)
+
+    // const result = await client.dag.get(CID.parse('QmY43r3bw9pJc28iFet42kAzVmtFuoa39kuxd4q4SG2gpz'))
+    // console.log(`result: `, result);
+    // for await (const data of client.cat('ipfs/QmY43r3bw9pJc28iFet42kAzVmtFuoa39kuxd4q4SG2gpz')) {
+    //   console.log(data)
```

Comment:
this was me testing the kubo-rpc-client directly.. something seems fishy with it being able to connect to the kubo node.